.github/workflows/ruby.yml

```diff
@@ -0,0 +1,43 @@
+name: Ruby
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        ruby: ['3.1']
+    env:
+      BUNDLE_GITHUB__COM: "${{ secrets.BUNDLE_GITHUB__COM }}"
+      BUNDLE_GEMS__CONTRIBSYS__COM: "${{ secrets.BUNDLE_GEMS__CONTRIBSYS__COM }}"
+      BUNDLE_WITHOUT: "production:qa:sandbox"
```

Comment:
You can actually delete this whole line.

Usually, we only need to set `BUNDLE_WITHOUT` for apps, not gems.

Reply:
Is the "apps, not gems" part because our apps tend to have a production group defined while the gems typically don't?